README.md

```diff
@@ -308,6 +308,99 @@ Descope SDK is automatically refreshes the session token when it is about to exp
 If the Descope project settings are configured to manage tokens in cookies.
 you must also configure a custom domain, and set it as the `baseUrl` prop in the `AuthProvider` component. See the above [`AuthProvider` usage](https://github.com/descope/react-sdk#wrap-your-app-with-auth-provider) for usage example.
 
+### Widgets
+
+Widgets are components that allow you to expose management features for tenant-based implementation. In certain scenarios, your customers may require the capability to perform managerial actions independently, alleviating the necessity to contact you. Widgets serve as a feature enabling you to delegate these capabilities to your customers in a modular manner.
+
+Important Note:
+
+- For the user to be able to use the widget, they need to be assigned the `Tenant Admin` Role.
+
+#### User Management
+
+The `User Management` widget will let you embed a user table in your site to view and change information, exposing the following fields:
+
+`LoginId`, `Status`, `Name`, `Email`, `Phone`, `Roles`
+The widget lets you:
+
+- Create a new user
+- Edit an existing user
+- Activate / disable an existing user
+- Reset an existing user's password
+- Remove an existing user's passkey
+- Delete an existing user
+
+Note:
+
+- Custom fields also appear in the table.
+
+###### Usage
+
+```js
+import { UserManagement } from '@descope/react-sdk';
+...
+	<UserManagement
+        widgetId="user-management-widget"
+		tenant="tenant-id"
+	/>
+```
+
+Example:
+[Manage Users](./examples/app/ManageUsers.tsx)
+
+#### Role Management
+
+The `Role Management` widget lets you delegate the role management to your customer.
+
+The widget lets you:
+
+- Create new role
+- Change an existing role's fields
+- Delete an existing role
+
+Note:
+
+- The `Editable` field is determined by the user's access to the role - meaning that project-level roles are not editable by tenant level users.
+- You need to pre-define the permissions that the user can use, which are not editable in the widget.
+
+###### Usage
+
+```js
+import { RoleManagement } from '@descope/react-sdk';
+...
+    <RoleManagement
+        widgetId="role-management-widget"
+		tenant="tenant-id"
+	/>
+```
+
+Example:
+[Manage Roles](./examples/app/ManageRoles.tsx)
+
+#### Access Key Management
+
+The `Access Key Management` widget lets you delegate the creation of access keys to your users and provide information on existing ones.
+
+The widget lets you:
+
+- Create an access key
```

Comment:
same here regarding consistency

Reply:
I've changed everything to be with - a or an

Reply:
ok, just don't forget the "new" for the access key, since you kept it for user and role.